exercises/practice/secret-handshake/.meta/example.cairo

```diff
@@ -0,0 +1,41 @@
+pub fn commands(number: u8) -> Array<ByteArray> {
+    let mut results: Array<ByteArray> = array![];
+
+    let mut i: u32 = 0;
+    for action in actions() {
+        if number.into() & (1 * pow(2, i)) != 0 {
+            results.append(action);
+        }
+        i += 1;
+    };
+
+    if number & 16 != 0 {
+        let mut reversed: Array<ByteArray> = array![];
+        let mut i = results.len();
```

Comment:
There's also a way to do this now using `while let` and popping from the back, removing the need for `i`.

```cairo
let mut span = results.span();
while let Option::Some(action) = span.pop_back() {
    reversed.append(action.clone());
}
```

You don't need to update this, just leaving it out there

Reply:
Awesome.